sockeye/training.py

```diff
@@ -311,13 +372,23 @@ def log_parameters(self):
         """
         arg_params, aux_params = self.module.get_params()
         total_parameters = 0
+        fixed_parameters = 0
+        learned_parameters = 0
         info = []  # type: List[str]
         for name, array in sorted(arg_params.items()):
             info.append("%s: %s" % (name, array.shape))
-            total_parameters += reduce(lambda x, y: x * y, array.shape)
+            num_parameters = reduce(lambda x, y: x * y, array.shape)
+            total_parameters += num_parameters
+            if name in self.module._fixed_param_names:
+                fixed_parameters += num_parameters
+            else:
+                learned_parameters += num_parameters
+        percent_fixed = 100 * (fixed_parameters / total_parameters)
```

Comment:
should we guard against `total_parameters = 0`? Admitted, its very unlikely :)

Reply:
Good catch!